extensions/notebook/src/test/book/book.test.ts

```diff
@@ -567,6 +571,35 @@ describe('BooksTreeViewTests', function () {
 			should(notebookUri.scheme).equal('untitled', 'Failed to get untitled uri of the resource');
 		});
 
+		it('openMarkdown should open markdown in the editor', async () => {
+			let executeCommandSpy = sinon.spy(vscode.commands, 'executeCommand');
+			let notebookPath = path.join(rootFolderPath, 'content', 'readme.md');
+			bookTreeViewProvider.openMarkdown(notebookPath);
+			should(executeCommandSpy.calledWith('markdown.showPreview')).be.true('openMarkdown should have called markdown.showPreview');
+			await vscode.commands.executeCommand('workbench.action.closeActiveEditor');
```

Comment:
Cleanup like this should either be in a finally, or as a afterEach so that we don't mess up other tests if these tests fail and leave stale state.

Reply:
You're currently calling `await vscode.commands.executeCommand('workbench.action.closeAllEditors');` in the afterAll - moving that to afterEach seems like a safer way of handling all this.